contributing/documentation/overview.rst

```diff
@@ -138,92 +173,27 @@ changes and push them:
-To do this, execute this command first to tell git about the original repository:
-
-.. code-block:: bash
-
-    $ cd projects/symfony-docs/
-    $ git remote add upstream https://github.com/symfony/symfony-docs.git
-
-Now you can **sync your fork** by executing the following command:
-
-.. code-block:: bash
-
-    $ cd projects/symfony-docs/
-    $ git fetch upstream
-    $ git checkout 2.3
-    $ git merge upstream/2.3
-
-This command will update the ``2.3`` branch, which is the one you used to
-create the new branch for your changes. If you have used another base branch,
-e.g. ``master``, replace the ``2.3`` with the appropriate branch name.
-
-Great! Now you can proceed by following the same steps explained in the previous
-section:
-
-.. code-block:: bash
-
-    # create a new branch to store your changes based on the 2.3 branch
-    $ cd projects/symfony-docs/
-    $ git checkout 2.3
-    $ git checkout -b my_changes
-
-    # ... do your changes
-
-    # submit the changes to your forked repository
-    $ git add xxx.rst     # (optional) only if this is a new content
-    $ git commit xxx.rst
-    $ git push origin my_changes
-
-    # go to GitHub and create the Pull Request
-    #
-    # Include this table in the description:
-    # | Q             | A
-    # | ------------- | ---
-    # | Doc fix?      | [yes|no]
-    # | New docs?     | [yes|no] (PR # on symfony/symfony if applicable)
-    # | Applies to    | [Symfony version numbers this applies to]
-    # | Fixed tickets | [comma separated list of tickets fixed by the PR]
-
-Your second contribution is now complete, so **go and celebrate again!**
-You can also see how your ranking improves in the list of
-`Symfony Documentation Contributors`_.
+**Step 10.** After your pull request is eventually accepted and merged in the
+Symfony documentation, you will be included in the `Symfony Documentation
+Contributors`_ list. Moreover, if you happen to have a `SensioLabsConnect`_
+profile, you will get a cool `Symfony Documentation Badge`_.
 
 Your Next Documentation Contributions
 -------------------------------------
 
-Now that you've made two contributions to the Symfony documentation, you are
-probably comfortable with all the Git-magic involved in the process. That's
-why your next contributions would be much faster. Here you can find the complete
-steps to contribute to the Symfony documentation, which you can use as a
-**checklist**:
+Your first contribution took some time because you had to fork the repository,
+learn how to write documentation, comply with the pull requests standards, etc.
+Now that you've completed your first contribution to the Symfony documentation,
+you are probably comfortable with all the Git-magic involved in the process.
+That's why your next contributions would be much faster. Here you can find the
+complete steps to contribute to the Symfony documentation, which you can use as
+a **checklist**:
```

Comment:
> Check you out! You've made your first contribution to the Symfony documentation!
> Somebody throw a party! Your first contribution took a little extra time because you
> needed to learn a few standards and setup your computer. But now, you're dangerous.
> 
> Here is a complete **checklist** of steps that will guide you through your next contribution
> to the Symfony docs: